Detections/Corelight/CorelightPossibleWebshellRarePOST.yaml

```diff
@@ -0,0 +1,36 @@
+id: db662e49-6e34-4d10-9d3c-5d04b5479658
+name: Corelight - Possible Webshell (Rare PUT or POST)
+description: |
+  'Detects rare post requests to a single webserver location.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated